README.md

```diff
@@ -25,7 +25,7 @@ application](https://github.com/juttle/juttle-viewer) and a full set of
 programs with rich charts and dynamic input controls in development or
 production.
 
-To learn more, see the [documentation site](https://juttle.github.io/juttle) to read about why juttle exists,
+Visit the [project website](http://juttle.github.io/) to see how juttle can fit into your stack. To learn more about the language, see the [documentation site](https://juttle.github.io/juttle) to read about why juttle exists,
```

Comment:
How about "Visit the project website to get an overview of the project and see how it can fit into your stack."